### ISSUE TITLE
Related post section on single post

### ISSUE THREAD

**Issue by vbaimas** [wplemon/gridd#45]
I really like the option to show some related posts on single post. 
Some inspiration : This section can have two options and be displayed as:

- **Grid layout.** Featured image, date, author, comments, description or not, post title.
- **List layout.** Appearing only the post title.

**Comment by aristath** [comment]
Related posts are plugin territory so they can't be in the theme.
However we could add styles for the most popular plugins...
I think Jetpack does related posts so we should definitely check that and make sure they are properly styled.

**Comment by vbaimas** [comment]
Xm. Instead of using plugin like Jetpack or something else, we can use a query to "call" the latest posts from the same category. What do you think?

**Comment by aristath** [comment]
It's possible and easy, but not allowed in themes hosted in the w.org repository because it's plugin territory.
So we can't add that functionality, we can just style plugins that add it.

**Comment by vbaimas** [comment]
Sorry, but I do not have a deep knowledge of the specific issue that you mentioned. It seems strange to me as I have seen it personally on some themes hosted in the w.org.

**Comment by aristath** [comment]
The review process in .org themes is a bit weird... There's the initial check when you submit a theme, and then once the theme is live you can do whatever you want. So some themes may be submitted without that feature and add it after they get approved.
These themes get sometimes reported by users and other developers for violating the guidelines and then they are forced to remove those features.
It's definitely possible to add it, but not that ethical 😅

**Comment by carolinan** [comment]
Who is saying that related posts are not allowed?
That is worth discussing in a theme review meeting in that case.

It's only showing posts withing the same category, excluding the current post. 
_I mean, it doesn't change or create any content, it displays existing content..._ 
Or did you mean something more advanced?

**Comment by aristath** [comment]
Yeah by "related posts" I had something more advanced in mind not just cat/tag. I was thinking of actual related content like some plugins do.
I suppose if we were to simply use the category/tags it would work just fine, it didn't even cross my mind 👍 
Marked as a feature request
